com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -988,7 +988,7 @@ void CreateCameraRenderTarget(ScriptableRenderContext context, ref RenderTexture
                     depthDescriptor.bindMS = depthDescriptor.msaaSamples > 1 && !SystemInfo.supportsMultisampleAutoResolve && (SystemInfo.supportsMultisampledTextures != 0);
 #endif
 
-                    depthDescriptor.bindMS |= depthDescriptor.msaaSamples > 1 && primedDepth && !SystemInfo.supportsMultisampleAutoResolve && (SystemInfo.supportsMultisampledTextures != 0);
+                    depthDescriptor.bindMS |= depthDescriptor.msaaSamples > 1 && !SystemInfo.supportsMultisampleAutoResolve && (SystemInfo.supportsMultisampledTextures != 0);
 
```

Comment:
this is the line of code to remove I mentioned in my comment (which I posted in the wrong place ops :) )